src/app/auth.service.ts

```diff
@@ -1,10 +1,15 @@
+import { HttpClient } from '@angular/common/http';
 import { Injectable } from '@angular/core';
 
 @Injectable({
   providedIn: 'root'
 })
 export class AuthService {
-  isLoggedIn: any;
 
-  constructor() { }
+  constructor(private http:HttpClient) { }
+  apiurl = "http://localhost:3500/users/"
+
+  public getUserByid(id:any){
+    return this.http.get(this.apiurl+id);
+  }
```

Comment:
Shouldn't it be on `Account.service.ts` man instead of `auth.service.ts`?

Reply:
let me check on it and return to you